requirements.txt

```diff
@@ -36,4 +37,4 @@ urllib3==1.23
 websocket-client==0.53.0
 whichcraft==0.5.2
 wrapt==1.10.11
-git+https://github.com/oxwhirl/smac.git
\ No newline at end of file
+git+https://github.com/oxwhirl/smac.git
```

Comment:
??? EOL or something I guess

[since it's the same on main](https://github.com/uoe-agents/epymarl/blob/main/requirements.txt#L39)